overlay/overrides.nix

```diff
@@ -130,8 +130,6 @@ in rec {
           (propagateEnv "pkg-config" [
             { name = "PKG_CONFIG_ALLOW_CROSS"; value = "1"; }
           ])
-        ];
-        propagatedNativeBuildInputs = drv.propagatedNativeBuildInputs or [ ] ++ [
```

Comment:
What about `drv.propagatedNativeBuildInputs`?

Reply:
Are you curious about what `drv.propagatedNativeBuildInputs` does? Or why the switch to `drv.propagatedBuildInputs` matters?